code/game/objects/obj_defense.dm

```diff
@@ -205,6 +205,8 @@ GLOBAL_DATUM_INIT(acid_overlay, /mutable_appearance, mutable_appearance('icons/e
 		var/turf/T = loc
 		if(T.intact && level == 1) //fire can't damage things hidden below the floor.
 			return
+	if(exposed_temperature < 300)
```

Comment:
Why would you do this?

Reply:
explain what your actual problem is

Reply:
i added this line because the cold fire would burn everything like a normal fire (including objects and you). it still burns you like normally and i cant figure out how to prevent that

Reply:
@iamgoofball I'm asking for clarification so I can understand the reasoning you clown